### ISSUE TITLE
Mention Autoloads on the style page

### ISSUE THREAD

**Issue by ShatteredRealityX** [godotengine/godot-docs#9521]
**Your Godot version:**
4.2.2

**Issue description:**
They aren't mentioned so users don't know the style for naming them (I guess they could follow the Autoload page?) Cuz I was looking at the code for an official demo project and the autoload there was in lower case

https://github.com/godotengine/godot-demo-projects/blob/4.2/networking/multiplayer_bomber/project.godot

**URL to the documentation page (if already existing):**
https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_styleguide.html

**Comment by AThousandShips** [comment]
This page doesn't touch on file names though does it? I don't think this is the right place for that

**Comment by ShatteredRealityX** [comment]
No, no the file name, the name you give it to be used in the script

Like you can do `Global.my_func()` instead of `global.my_func()`

**Comment by AThousandShips** [comment]
My bad it does mention to use `PascalCase` for class names, and `snake_case` for the files

I don't think autoloads belongs explicitly in this guide, they're not a feature of GDScript but the editor itself, so their naming convention should be covered elsewhere IMO

**Comment by ShatteredRealityX** [comment]
okay maybe it can be mentioned on the singletons page?
